new/LIT5898DersanaMikael.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5898DersanaMikael" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+              <title xml:lang="en" corresp="#t1">Homily on the Archangel Michael</title>
```

Comment:
same as above. I think we should stick to the guidelines title structuring (Geez is t1, etc.). Then the title we will want to have printed can be added with type="main"